extensions/cdc-mysql/src/test/java/com/hazelcast/jet/cdc/mysql/AbstractMySqlCdcIntegrationTest.java

```diff
@@ -50,8 +50,10 @@ public abstract class AbstractMySqlCdcIntegrationTest extends AbstractCdcIntegra
                     .withPassword("mysqlpw")
     );
 
-    @Before
-    public void ignoreOnJdk15OrHigher() {
+    @BeforeClass
+    public static void ignoreOnArm64OrOnJdk15OrHigher() {
+        //There is no arm64 version of currently used docker image
+        assumeNoArm64Architecture();
```

Comment:
IMHO It's better to upgrade image to 1.9.6, which has ARM images

Reply:
I have created a task to update debezium separately: https://github.com/hazelcast/hazelcast/issues/23491
Currently, we are using `<debezium.version>1.9.5.Final</debezium.version>` versions should be aligned I think.

Reply:
that should not matter, should work just fine when upgrading to minor version

Reply:
I updated image of mysql, but for postgres there is no working version (even if in theory supports arm64).